src/datastructures.apib

```diff
@@ -59,6 +59,12 @@
 + number: `1` (number, optional)
     + Default: `1`
 
+## Pagination
++ count: `200` (number)
++ per_page: `10` (number)
++ current_page: `2` (number)
++ total_pages: `20` (number)
```

Comment:
The whole pagination block is optional. We will only include it for the endpoints where we count the total number of elements in the collection (currently nowhere, the first one will be project items if we agree on this form of pagination).

Reply:
What I meant is that in other data structures in this file we explicitly define which fields are required and which are optional. Should we not do the same here?

Reply:
EDIT: I see we don't do it for all the data structure but maybe we should.

Reply:
@regularjack I would like to discuss this on Slack, to better understand your concern. However, I will merge the PR for now since it has a majority already and if we do find, on Slack, that there needs to be something changed, I will create a follow-up PR.